libs/martech/helpers.js

```diff
@@ -0,0 +1,494 @@
+const TARGET_TIMEOUT_MS = 4000;
```

Comment:
This will introduce a new network call and blocks the rendering pipeline pre LCP. It's likely the opposite of what you want by trying to make the pipeline more efficient

Reply:
@mokimo this file has the interact call which earlier was in launch and now is moved to milo, that enables a faster load of a few dependant scripts. Since all these to begin were pre lcp.